lib/actions/auth.coffee

```diff
@@ -177,5 +178,15 @@ exports.whoami =
 		resin.auth.whoami().then (username) ->
 			if not username?
 				throw new Error('Username not found')
-			console.log(username)
+			resin.auth.getEmail().then (email) ->
+				if not email?
```

Comment:
I don't think this check is necessary. I believe a user always has an email, and even if no, `visuals`'s table will add a blank space at the side of the EMAIL property, making it clear that there is no email.